### ISSUE TITLE
failed to clone code repository in the Pipeline

### ISSUE THREAD

**Issue by jettjia** [kubesphere/ks-devops#445]
1.安装用的kubekey安装的，节点是：1个master, 1个node；工作节点把master和node都设置成了 worker
2.k8s版本 1.21, kubesphere版本3.2.1
3.kubesphere的Jenkinsfile配置如下：
```groovy
pipeline {
  agent {
    node {
      label 'go'
    }

  }
  stages {
    stage('拉取代码') {
      agent none
      steps {
        container('base') {
          git(url: 'https://gitee.com/jidi/cicd.git', credentialsId: 'gitee-id', branch: 'master', changelog: true, poll: false)
          sh 'ls'
        }

      }
    }

  }
}
```

4.编辑截图如下
 
![image](https://user-images.githubusercontent.com/25764003/149250450-d6a93b38-6a56-4ce6-b9fe-38feee3d3b67.png)

错误如下：
1.点击运行没有反应，状态都是未运行
2.而且每一次运行的时间都被改成一样了
 [
![image](https://user-images.githubusercontent.com/25764003/149248588-e0cc432e-fe54-48e7-940c-182edd9c51f9.png)
](url)

**Comment by LinuxSuRen** [comment]
I'm wondering if you have checked the Jenkins agent pods. The image pulling might need some time.

**Comment by LinuxSuRen** [comment]
This is a known issue in v3.2.1. We have fixed it in v3.3.0
/close
